app/models/header_lookup.rb

```diff
@@ -10,18 +10,22 @@ class HeaderLookup
   MESH_ID_URI = "https://id.nlm.nih.gov/mesh/"
   MEMOIZED_MESH_FILE = "memoized_mesh.txt"
   MEMOIZED_LCSH_FILE = "memoized_lcsh.txt"
+  MEMOIZED_LCNAF_FILE = "memoized_lcnaf.txt"
   SEARCHABLE_MESH_FILE = "subjects_mesh.yml"
   SEARCHABLE_LCSH_FILE = "subjects_lcsh.yml"
+  SEARCHABLE_LCNAF_FILE = "trimmed_lcnaf.yml"
```

Comment:
Should this be `updated_trimmed_lcnaf.yml`?

Reply:
Nah, I should change it in the script that was building that. I was keeping two files at first so I could check the differences, but @fenekku's suggestions worked well so I removed the original file and renamed "updated_trimmed_lcnaf.yml" to "trimmed_lcnaf.yml"